service/src/main/kotlin/fi/espoo/evaka/reservations/ReservationControllerCitizen.kt

```diff
@@ -0,0 +1,77 @@
+package fi.espoo.evaka.reservations
+
+import fi.espoo.evaka.Audit
+import fi.espoo.evaka.shared.ChildId
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.auth.UserRole
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.domain.BadRequest
+import fi.espoo.evaka.shared.domain.FiniteDateRange
+import fi.espoo.evaka.shared.domain.HelsinkiDateTime
+import fi.espoo.evaka.shared.domain.isWeekend
+import org.jdbi.v3.core.kotlin.mapTo
+import org.jdbi.v3.json.Json
+import org.springframework.format.annotation.DateTimeFormat
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.util.UUID
+
+@RestController
+class ReservationControllerCitizen {
+    @GetMapping("/citizen/reservations")
+    fun getReservations(
+        db: Database.Connection,
+        user: AuthenticatedUser,
+        @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) from: LocalDate,
+        @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) to: LocalDate,
+    ): List<DailyReservationData> {
+        Audit.AttendanceReservationRead.log(targetId = user.id)
```

Comment:
Should you somehow specify that it's a guardian reading their own children's reservations? Employees need a pretty much identical audit event.